centrallix/objectsystem/obj_datatypes.c

```diff
@@ -1401,7 +1414,7 @@ objDataToMoney(int data_type, void* data_ptr, pMoneyType m)
     char* endptr2;
     double dbl;
     int is_neg = 0;
-    unsigned long intval;
+    long long intval;
```

Comment:
double check intval isn't used elsewhere in the function with the assumption that it's signed/possibly fewer bits??